service/history/historyEngine.go

```diff
@@ -2536,6 +2537,18 @@ func (e *historyEngineImpl) NotifyNewTransferTasks(
 	}
 }
 
+func (e *historyEngineImpl) NotifyNewVisibilityTasks(
+	tasks []persistence.Task,
+) {
+
+	if len(tasks) > 0 {
+		// TODO (alex): add visibility processor
+		// task := tasks[0]
+		// clusterName := e.clusterMetadata.ClusterNameForFailoverVersion(task.GetVersion())
+		// e.visibilityProcessor.NotifyNewTask(clusterName, tasks)
```

Comment:
FYI, visibility task processing is local only, meaning pushing something to local ES cluster / local archival endpoint

Reply:
Thanks for head up.